### PR TITLE
Added missing translation for login-config-totp template in spanish

### PR DESCRIPTION
**Pull request by jaime-pineiro-imatia**

### CODE REVIEW

**Comment by jonkoops** [comment]
Hi @jaime-pineiro-imatia, thanks for your PR! We appreciate you taking the time to contribute this. If possible, could you get another Spanish native speaker to review your work?

File: themes/src/main/resources-community/theme/base/login/messages/messages_es.properties [hunk 1]
```diff
@@ -68,10 +68,23 @@ country=Pa\u00EDs
...
 gssDelegationCredential=GSS Delegation Credential
 
+loginTotpIntro=Es necesario configurar un generador de claves de un sólo uso para acceder a esta cuenta
 loginTotpStep1=Instala <a href=\"https://freeotp.github.io/\" target=\"_blank\">FreeOTP</a> o Google Authenticator en tu tel\u00E9fono m\u00F3vil. Ambas aplicaciones est\u00E1n disponibles en <a href=\"https://play.google.com\">Google Play</a> y en la App Store de Apple.
 loginTotpStep2=Abre la aplicaci\u00F3n y escanea el c\u00F3digo o introduce la clave.
 loginTotpStep3=Introduce el c\u00F3digo \u00FAnico que te muestra la aplicaci\u00F3n de autenticaci\u00F3n y haz clic en Enviar para finalizar la configuraci\u00F3n
+loginTotpStep3DeviceName=Introduce un nombre de dispositivo para que te ayude a gestionar tus dispositivos OTP.
+loginTotpManualStep2=Abre la aplicación e introduce la clave:
+loginTotpManualStep3=Usa los siguientes parámetros de configuración si la aplicación te permite introducirlos:
+loginTotpUnableToScan=¿Tienes problemas para escanear?
```
Comment (ikhomyn) [inline themes/src/main/resources-community/theme/base/login/messages/messages_es.properties:78]:
```suggestion
loginTotpUnableToScan=¿No consigues escanear?
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_es.properties [hunk 2]
```diff
@@ -68,10 +68,23 @@ country=Pa\u00EDs
...
+loginTotpScanBarcode=¿Escanear el código de barras?
+loginCredential=Credenciales
 loginOtpOneTime=C\u00F3digo de un solo uso
+loginTotpType=Tipo
+loginTotpAlgorithm=Algoritmo
+loginTotpDigits=Dígitos
+loginTotpInterval=Intervalo
+loginTotpCounter=Contador
+loginTotpDeviceName=Nombre de dispositivo
```
Comment (ikhomyn) [inline themes/src/main/resources-community/theme/base/login/messages/messages_es.properties:87]:
```suggestion
loginTotpDeviceName=Nombre del dispositivo
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_es.properties [hunk 3]
```diff
@@ -68,10 +68,23 @@ country=Pa\u00EDs
 emailVerified=Email verificado
 gssDelegationCredential=GSS Delegation Credential
 
+loginTotpIntro=Es necesario configurar un generador de claves de un sólo uso para acceder a esta cuenta
 loginTotpStep1=Instala <a href=\"https://freeotp.github.io/\" target=\"_blank\">FreeOTP</a> o Google Authenticator en tu tel\u00E9fono m\u00F3vil. Ambas aplicaciones est\u00E1n disponibles en <a href=\"https://play.google.com\">Google Play</a> y en la App Store de Apple.
 loginTotpStep2=Abre la aplicaci\u00F3n y escanea el c\u00F3digo o introduce la clave.
 loginTotpStep3=Introduce el c\u00F3digo \u00FAnico que te muestra la aplicaci\u00F3n de autenticaci\u00F3n y haz clic en Enviar para finalizar la configuraci\u00F3n
+loginTotpStep3DeviceName=Introduce un nombre de dispositivo para que te ayude a gestionar tus dispositivos OTP.
+loginTotpManualStep2=Abre la aplicación e introduce la clave:
+loginTotpManualStep3=Usa los siguientes parámetros de configuración si la aplicación te permite introducirlos:
```
Comment (ikhomyn) [inline themes/src/main/resources-community/theme/base/login/messages/messages_es.properties:77]:
```suggestion
loginTotpManualStep3=Usa los siguientes parámetros de la configuración si la aplicación te permite introducirlos:
```

File: themes/src/main/resources-community/theme/base/login/messages/messages_es.properties [hunk 4]
```diff
@@ -68,10 +68,23 @@ country=Pa\u00EDs
 emailVerified=Email verificado
 gssDelegationCredential=GSS Delegation Credential
 
+loginTotpIntro=Es necesario configurar un generador de claves de un sólo uso para acceder a esta cuenta
 loginTotpStep1=Instala <a href=\"https://freeotp.github.io/\" target=\"_blank\">FreeOTP</a> o Google Authenticator en tu tel\u00E9fono m\u00F3vil. Ambas aplicaciones est\u00E1n disponibles en <a href=\"https://play.google.com\">Google Play</a> y en la App Store de Apple.
 loginTotpStep2=Abre la aplicaci\u00F3n y escanea el c\u00F3digo o introduce la clave.
 loginTotpStep3=Introduce el c\u00F3digo \u00FAnico que te muestra la aplicaci\u00F3n de autenticaci\u00F3n y haz clic en Enviar para finalizar la configuraci\u00F3n
+loginTotpStep3DeviceName=Introduce un nombre de dispositivo para que te ayude a gestionar tus dispositivos OTP.
```
Comment (ikhomyn) [inline themes/src/main/resources-community/theme/base/login/messages/messages_es.properties:75]:
```suggestion
loginTotpStep3DeviceName=Introduce un nombre del dispositivo para que te ayude a gestionar tus dispositivos OTP.
```

**Review by ikhomyn — APPROVED** [review]

File: themes/src/main/resources-community/theme/base/login/messages/messages_es.properties (hunk 3 above)
Comment (jaime-pineiro-imatia) [reply themes/src/main/resources-community/theme/base/login/messages/messages_es.properties:77]: I don't think this translation is correct

File: themes/src/main/resources-community/theme/base/login/messages/messages_es.properties (hunk 4 above)
Comment (jaime-pineiro-imatia) [reply themes/src/main/resources-community/theme/base/login/messages/messages_es.properties:75]: I don't think this translation is correct

**Review by miquelsi — APPROVED** [review]

**Review by jonkoops — APPROVED** [review]

**Review by ssilvert — APPROVED** [review]

**Pull request merged**